### ISSUE TITLE
Standard color palettes for different uses.

### ISSUE THREAD

**Issue by omuelle** [mbannert/tstools#179]
For example like [here](http://colorbrewer2.org/#type=qualitative&scheme=Paired&n=6). Would be nice to have, though low priority.

**Comment by mbannert** [comment]
hmm actually we have standard palettes for line colors, fill colors etc. They should be selected according to ETH / KOF color schemes. Doesn't `init_tsplot_theme()` and the resulting parameter list contain what you're asking for. 

Still though I agree, a color blind palette and other special palette would be nice.

**Comment by omuelle** [comment]
There are only 7 colors so the rest appears white, also the bar fill colors aren't easily distinguishable. The line colors are better in that regard though.

**Comment by mbannert** [comment]
agreed. It's based on the ETH layout colors, but you're right, this needs some work!

**Comment by HomoCodens** [comment]
Anybody is free to create their own palettes and possibly submit them here for inclusion in  future releases.

How would we (conveniently) provide those?
For instance the line, bar and sum line colors would probably all be part of the same color palette but they are separate theme parameters.

**Comment by omuelle** [comment]
I was thinking of a function that returns some standard palettes. For example, in the app I linked to there are color schemes for sequential, diverging and qualitative data. In any case, there are probably already decent R packages for that purpose, I haven't looked.

**Comment by mbannert** [comment]
@omuelle yup there are quite some R packages for this. Yet looking for one of them with low dependencies themselves to make tstools *suggest* it would be one solution. Another idea is just to borrow some concepts and have a lightweight color palette function in tstools, too. 

I like your idea of having color palettes for different purposes. This fits the idea of tstools very well: someone with a purpose in mind, like: I want a stacked bar chart with some gradient type of style OR I want 6 lines in one graph which are clearly distinguishable.  We could pick this mindset up by having palette functions (or one function with parameters). Still, we the tsplot argument should simple accept a color vector. 

Bottom line: let's keep this open in order to keep it in mind. Currently not a high priority. 
Probably Layout can help us here, defining palettes for purposes. I'll talk to them.

**Comment by cboulanger** [comment]
Old thread but I wonder if anyone has meanwhile created some alternative color schemes. The built-in one is visually pleasing but hard to distinguish on bar charts.
